HousingRepairsOnlineApi/Helpers/IIdGenerator.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace HousingRepairsOnlineApi.Helpers
+{
+    public abstract class IIdGenerator
```

Comment:
This should be an `interface` as it doesn't provide any implementation and also doesn't serve as the base of the class hierarchy.